### PR TITLE
UX: unified About / Privacy / ToS / FAQ topnav

### PR DESCRIPTION
**Pull request by techAPJ**
This PR encapsulates 2 changes:

* [unified About / Privacy / ToS / FAQ topnav](https://github.com/techAPJ/discourse/commit/ddb8378ab049e46200b76ad80a672cc0cc50e3d6)

* [show real name(s) on the about page](https://github.com/techAPJ/discourse/commit/5e8938c7b613a6a9d2c72c09947745bf32cd0822)

Screenshot:

screenshot from 2015-02-23 20 22 24

[Relevant post on meta](https://meta.discourse.org/t/disclaimer-section-on-the-about-page/25461/13?u=techapj)

### CODE REVIEW

**Comment by techAPJ** [comment]
Build failure seems unrelated:

```
Unexpected error while processing request: No route matches [GET] "/user_avatar/supermathie/20/5247.png"
```

File: app/assets/javascripts/discourse/controllers/about.js.es6 [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import ObjectController from 'discourse/controllers/object';
 
 export default ObjectController.extend({
+  faqOverriden: function() {
+    return !(this.siteSettings.faq_url.length === 0);
```
Comment (eviltrout) [inline app/assets/javascripts/discourse/controllers/about.js.es6]: Couldn't this be done as `Ember.computed.gt('siteSettings.faq_url.length', 0)`? I think that should work.

File: app/assets/javascripts/discourse/templates/components/user-small.hbs [hunk 2]
```diff
@@ -1,4 +1,7 @@
 {{#link-to 'user' user.username}}
   {{avatar user imageSize="tiny"}}
   {{user.username}}
+  {{#if user.name}}
```
Comment (eviltrout) [inline app/assets/javascripts/discourse/templates/components/user-small.hbs]: Why the `#if` statement? if `user.name` is blank wouldn't it just output nothing?

Comment (techAPJ) [inline app/assets/javascripts/discourse/templates/components/user-small.hbs]: Yes, but that would leave empty bracket `({{user.name}})`

Comment (eviltrout) [inline app/assets/javascripts/discourse/templates/components/user-small.hbs]: Oh I see, that's cool then.

File: app/assets/javascripts/discourse/controllers/about.js.es6 (hunk 1 above)
Comment (techAPJ) [inline app/assets/javascripts/discourse/controllers/about.js.es6]: Nice! Updated the PR.

**Comment by eviltrout** [comment]
Thanks!

**Pull request merged**